DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/use_library/part1_native/src/use_native.cpp

```diff
@@ -5,20 +5,47 @@
 // =============================================================
 #include <sycl/sycl.hpp>
 #include <sycl/ext/intel/fpga_extensions.hpp>
-#include "lib.hpp"
 #include "exception_handler.hpp"
 
+#include <sycl/ext/intel/ac_types/ac_int.hpp>
+#include <sycl/ext/intel/experimental/pipe_properties.hpp>
+#include <sycl/ext/intel/experimental/pipes.hpp>
+#include <sycl/ext/intel/prototype/interfaces.hpp>
```

Comment:
review kevin's interfaces overview code sample; the header file list should be different:

```
#include <sycl/ext/intel/ac_types/ac_int.hpp>
// oneAPI headers
#include <sycl/ext/intel/fpga_extensions.hpp>
#include <sycl/ext/intel/experimental/fpga_kernel_properties.hpp>
#include <sycl/ext/oneapi/annotated_arg/annotated_arg.hpp>
#include <sycl/sycl.hpp>

#include "exception_handler.hpp"
```

Reply:
is this still valid right now? I believe sycl.hpp has comes first to prevent compiler crash

Reply:
that's an old issue that is fixed in 2024.0